pkg/v1/remote/descriptor.go

```diff
@@ -237,6 +239,49 @@ func (f *fetcher) url(resource, identifier string) url.URL {
 	}
 }
 
+// https://github.com/opencontainers/distribution-spec/blob/main/spec.md#referrers-tag-schema
+func fallbackTag(d name.Digest) name.Tag {
+	return d.Context().Tag(fmt.Sprintf("%s-%s", d.Digest().Algorithm, d.Digest().Hex))
```

Comment:
I think I wrote this, but maybe we should just do:

```
return d.Context().Tag(strings.Replace(d.DigestStr(), ":", "-", 1))
```

This also lets us drop the new `d.Digest()` method in `pkg/name`.